content/pt-br/docs/reference/access-authn-authz/authorization.md

```diff
@@ -0,0 +1,248 @@
+ * **Recurso** - O identificador ou nome do recurso que está sendo acessado (somente para requisições de recursos) -- Para requisições de recursos usando os verbos `get`, `update`, `patch` e `delete`, deve-se fornecer o nome do recurso.
+ * **Subrecurso** - O sub-recurso que está sendo acessado (somente para solicitações de recursos).
+ * **Namespace** - O namespace do objeto que está sendo acessado (somente para solicitações de recursos com namespace).
+ * **Grupo de API** - O {{< glossary_tooltip text="API Group" term_id="api-group" >}} sendo acessado (somente para requisições de recursos). Uma string vazia designa o _core_ [Grupo de API](/docs/reference/using-api/#api-groups).
+
+## Determine o verbo da requisição {#determine-the-request-verb}
+
+**Requisições de não-recursos**
+
+Requisições para endpoints diferentes de `/api/v1/...` ou `/apis/<group>/<version>/...`
+são considerados "requisições sem recursos" e usam o método HTTP em letras minúsculas da solicitação como o verbo.
+Por exemplo, uma solicitação `GET` para endpoints como `/api` ou `/healthz` usaria `get` como o verbo.
+
+**Requisições de recursos**
+Para determinar o verbo de requisição para um endpoint de API de recurso, revise o verbo HTTP
+utilizado e se a requisição atua ou não em um recurso individual ou em uma
+coleção de recursos:
+
+Verbo HTTP | Verbo de Requisição
+---------- |---------------
+POST       | create
+GET, HEAD  | get (para recursos individuais), list (para coleções, includindo o conteúdo do objeto inteiro), watch (para assistir um recurso individual ou coleção de recursos)
+PUT        | update
+PATCH      | patch
+DELETE     | delete (para recursos individuais), deletecollection (para coleções)
+
+Às vezes, o Kubernetes verifica a autorização para permissões adicionais utilizando verbos especializados. Por exemplo:
+
+* [PodSecurityPolicy](/docs/concepts/security/pod-security-policy/)
+  * `use` verbo em recursos `podsecuritypolicies` no grupo `policy` de API.
+* [RBAC](/docs/reference/access-authn-authz/rbac/#privilege-escalation-prevention-and-bootstrapping)
+  * `bind` e `escalate` verbos em `roles` e recursos `clusterroles` no grupo `rbac.authorization.k8s.io` de API.
+* [Authentication](/pt-br/docs/reference/access-authn-authz/authentication/)
+  * `impersonate` verbo em `users`, `groups`, e `serviceaccounts` no grupo de API principal, e o `userextras` no grupo `authentication.k8s.io` de API.
+
+## Modos de Autorização {#authorization-modules}
+
+O servidor da API Kubernetes pode autorizar uma solicitação usando um dos vários modos de autorização:
+
+ * **Node** - Um modo de autorização de finalidade especial que concede permissões a `kubelets` com base nos `pods` que estão programados para execução. Para saber mais sobre como utilizar o modo de autorização do nó, consulte [Node Authorization](/docs/reference/access-authn-authz/node/).
```

Comment:
It's OK to not format "kubelet" or "Pod" as code. It's better to render Pod with a capital P, as a convention to refer to Kubernetes resources.